iOS/Projects/Features/Record/Sources/ExerciseSelectScene/View/ExerciseCardCell.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  ExerciseCardCell.swift
+//  RecordFeature
+//
+//  Created by MaraMincho on 11/16/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import DesignSystem
+import UIKit
+
+// MARK: - ExerciseCardCell
+
+class ExerciseCardCell: UICollectionViewCell {
+  static let identifier = "ExerciseCardCell"
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    makeShaodwAndRounded()
```

Comment:
p1: 메서드에 오타가 났습니다. ㅎㅎ
```suggestion
    makeShadowAndRounded()
```

Reply:
확인했습니다~